test/algorithm.cpp

```diff
@@ -75,6 +77,16 @@ void test_algorithm() {
             BULK_CHECK(bulk::min(xs) == 1, "min of coarray");
             BULK_CHECK(bulk::sum(xs) == 2 * p * (2 * p + 1) / 2,
                        "sum of coarray ");
+
+            BULK_CHECK(bulk::sum(world, (std::int64_t)std::numeric_limits<std::int32_t>::max()) ==
```

Comment:
To be consistent with the rest of the code base, I don't qualify fixed-width integer types with `std::`. Would be great if you could make this simply `int64_t` and `int32_t` 👍

Reply:
I removed the prefixes and the include for cstdint, but I actually don't really understand where the constants are coming from now :grimacing: (but it works)